src/twister2/fixtures/builder.py

```diff
@@ -18,7 +18,7 @@
 
 
 @pytest.fixture(scope='function')
-def builder(request: pytest.FixtureRequest) -> Generator[BuilderAbstract, None, None]:
+def build_manager(request: pytest.FixtureRequest) -> Generator[BuildManager, None, None]:
```

Comment:
There is duplicated name `build_manger` in line 44. I think you can implemented fixture like this:
```python
@pytest.fixture(name='build_manager', scope='function')
def fixture_build_manager(request: pytest.FixtureRequest):
   ...
```
(I notice `pylint` complains about this for `builder` fixture)

Reply:
I must re-configure pylint :) Thanks for comment. I changes it.